src/js/components/AirPlayButton.js

```diff
@@ -89,12 +89,17 @@ AirPlayButton = {
          return;
       }
 
-      mediaEl.addEventListener('webkitplaybacktargetavailabilitychanged', function(event) {
+      function onTargetAvailabilityChanged(event) {
          if (event.availability === 'available') {
             self.show();
          } else {
             self.hide();
          }
+      }
+
+      mediaEl.addEventListener('webkitplaybacktargetavailabilitychanged', onTargetAvailabilityChanged, this);
```

Comment:
One other thing I missed in the previous review: The third parameter to the `addEventListener` method is either an options object or a boolean flag, not a reference to `this`: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

We should just remove the third parameter here and below. The `onTargetAvailabilityChanged` handler has access to the proper `this` via the `self` variable.

Reply:
Fixed. 👍  Thanks!